templates/customernotification.html

```diff
@@ -68,6 +68,9 @@
                         <br>
                 {{ end }}
                 <ul>
+                    {{if eq .Name "clusterid" }}
+                        <li>Cluster ID: {{ .Value }}</li>
+                    {{ end }}
```

Comment:
I think we should provide the clusterID in the start of email.
Based on your change, I think if we have multiple PVCs then it will print clusterID multiple times.

Reply:
This will break the email template, suppose we have the same alert for diff cluster/namespace
Acc to you, this will look like:

- clusterID1
- alertmessage

- pvc name1

- clusterID2
- pvc name2

Reply:
Customers will always receive separate mail for different clusters.